ikasaneip/component/validator/pom.xml

```diff
@@ -47,6 +47,12 @@
             <artifactId>jaxb-impl</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.xmlresolver</groupId>
+            <artifactId>xmlresolver</artifactId>
+            <version>4.4.1</version>
```

Comment:
Should we not pull the version out to the parent pom.xml of the proejct with all other version management?